gazebo_answers_tracker.py

```diff
@@ -0,0 +1,97 @@
+import requests
+from bs4 import BeautifulSoup
```

Comment:
I think that we are going to need a INSTALL document somehow. I added bs4 through `pip3` but get the following error:
```
Entries sorted based on votes: 
Page 1

Question #1
Keywords of question title are:  ('correct way',)

Traceback (most recent call last):
  File "gazebo_answers_tracker.py", line 72, in <module>
    soup_rss = BeautifulSoup(rss.text, "xml")
  File "/home/jrivero/.local/lib/python3.6/site-packages/bs4/__init__.py", line 196, in __init__
    % ",".join(features))
bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: xml. Do you need to install a parser library?
```

Reply:
Oh I see. I didn't face this issue on my Ubuntu instance, but just checked that it happened on my Mac.

The reason for this is that BS4 needs an external parser to parse the XML content. The `lxml` library is required for this. 
You can do
```
pip install lxml
```
or any of the one's mentioned [here](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser).

I'll add the installation document after the current implementation is approved and merged (if required).